networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -78,12 +83,16 @@ def dfs_edges(G, source=None, depth_limit=None):
     if depth_limit is None:
         depth_limit = len(G)
 
+    get_children = G.neighbors
+    if sort_neighbors is not None:
+        get_children = lambda node: iter(sort_neighbors(G.neighbors(node)))
+
```

Comment:
Definitely not a blocker, but you could put the assignment all into one expression to avoid the multiple assignments, something like:

```suggestion
    get_children = G.neighbors if sort_neighbors is None else lambda n: iter(sort_neighbors(G.neighbors(n)))

```